### ISSUE TITLE
Remove Disconnect and Google mapping references

### ISSUE THREAD

**Issue by skim1102** [mozilla-services/shavar-list-creation#118]
# About this Issue
As mentioned in this [issue](https://github.com/mozilla-mobile/firefox-ios/issues/5968) we are not longer using `google-mapping.json`. Also, since we applied all the mapping for Disconnect category since [this pr](https://github.com/mozilla-services/shavar-prod-lists/pull/110), we can now remove all the references to `google-mapping.json`, Disconnect category, and `disconnect-mapping.json`

# Acceptance Criteria
- [ ] Remove all the references to `google-mapping.json`
- [ ] Remove all the references to Disconnect category. See https://github.com/mozilla-services/shavar-list-creation/issues/126 for more details
- [x] Remove all the references to `disconnect-mapping.json`
- [x] Remove all references to Disconnect category in [config](https://github.com/mozilla-services/shavar-list-creation-config/)

**Comment by boolean5** [comment]
Hi, @skim1102! May I work on this issue?

**Comment by skim1102** [comment]
@boolean5, definitely!

**Comment by boolean5** [comment]
I  just submitted a PR that removes references to the Disconnect category and `disconnect_mapping.json`. @skim1102, could you please take a look?
As far as I can see there are no references to `google_mapping.json` in this repo. Am I missing something?

**Comment by skim1102** [comment]
@boolean5, if you follow the PR and comments in https://github.com/mozilla-services/shavar-list-creation/pull/70 it seems `google_mapping.json` was folded into `disconnect-mapping.json` as mentioned [here](https://github.com/mozilla-services/shavar-list-creation/pull/72/files#diff-f1f16ae8cb1fa77b473aff7c3793d391L1). I also did a global search in shavar-list-creation code base and was unable to see any reference to `google_mapping.json` as well. Thanks for looking into that (checking off the A/C!).

**Comment by skim1102** [comment]
Waiting for response from https://github.com/mozilla-mobile/firefox-ios/issues/5968#issuecomment-598410933 to remove `disconnect_mapping.json` file.

**Comment by skim1102** [comment]
@Amejia481, are all the android projects not using the GeckoView no longer using `google_mapping.json` and `disconnect_mapping.json`? I just want to confirm that those two files can be removed now.

**Comment by Amejia481** [comment]
Yes @skim1102, we removed on https://github.com/mozilla-mobile/android-components/pull/6183/commits/0ef4a7b16214fb824fdbe61c3148a82942feebe6. From our side, It is OK to remove it.

**Comment by boolean5** [comment]
>Waiting for response from mozilla-mobile/firefox-ios#5968 (comment) to remove disconnect_mapping.json file.

@skim1102, my understanding from the conversation in https://github.com/mozilla-mobile/firefox-ios/issues/5968 and from a quick look at the Firefox-iOS code base is that it uses the `google_mapping.json` file from the shavar-prod-lists repository. So `disconnect_mapping.json` in shavar-list-creation is not used anymore and can be safely removed, right?

**Comment by skim1102** [comment]
@boolean5, yes--`disconnect_mapping.json` can be removed and there should be another PR to remove disconnect_mapping.json if the firefox-ios issue is not closed before the merging of #121. Thanks for reading the thread on the firefox-ios issue.

**Comment by skim1102** [comment]
Unchecked the first a/c in the PR description because firefox-ios still uses the `google_mapping.json`

**Comment by skim1102** [comment]
Checked

> Remove all references to Disconnect category in config

after merging https://github.com/mozilla-services/shavar-list-creation-config/pull/60

**Comment by skim1102** [comment]
Re-opening because

> Remove all the references to Disconnect category. 

is still not fulfilled yet.

**Comment by skim1102** [comment]
https://github.com/mozilla-mobile/firefox-ios/issues/5968#issuecomment-615432530
`google-mapping.json` can be removed!

**Comment by boolean5** [comment]
Great! I submitted a PR for this here: https://github.com/mozilla-services/shavar-prod-lists/pull/176
